docs/pages/application-access/getting-started.mdx

```diff
@@ -68,29 +79,124 @@ $ tctl tokens add \
     --app-uri=http://localhost:3000
 ```
 
-### Start Teleport
+### Configure and Start Teleport
 
 
 Install Teleport on the host where you will run the Teleport Application Service. See our [Installation](../installation.mdx) page for options besides Linux servers.
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Now start Teleport and point it to the application endpoint:
+<Tabs>
+<TabItem label="Using a config file">
+Create the Teleport Application Service configuration at `/etc/teleport.yaml`. Replace <Var name="teleport.example.com" /> with
+your Teleport Proxy address (`teleport.example.com`) or Teleport Cloud tenant (e.g. `mytenant.teleport.sh`).
+
+```code
+$ sudo teleport configure \
+  -o file \
+  --proxy=<Var name="teleport.example.com" />:443 \
+  --token=/tmp/token \
+  --roles=app \
+  --app-name=grafana \
+  --app-uri=http://localhost:3000
+```
+  
+Make sure to update `--app-name` and `--app-uri` accordingly if you're using your own web application.
+
+The `--token` flag points to the file on the Application Service host where we stored the token that we generated earlier.
+
+(!docs/pages/includes/start-teleport.mdx service="the Teleport Application Service"!)
+</TabItem>
+<TabItem label="With CLI flags">
+
+You can start the Teleport Application Service without configuration file using a
```

Comment:
Can we switch to `teleport configure` + `teleport start` instead of having them start teleport without a configuration file?

Reply:
So we want to remove the cli flags teleport app start ?

Reply:
See latest update @zmb3 .  Just configure and start with systemctl